LayoutTests/http/tests/misc/heic-accept-header.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html><!-- webkit-test-runner [ additionalSupportedImageTypes=public.heic;public.heics ] -->
```

Comment:
I think this test should be enabled only for macOS and iOS only since additionalSupportedImageTypes() is defined for Cocoa ports. And other ports do not support HEIC.

Reply:
🆗